### ISSUE TITLE
iOS Support

### ISSUE THREAD

**Issue by elifoster** [Esteemed-Innovation/Flaxbeards-Steam-Power-Mobile#2]
There should be an iOS version of the mod. Additionally, they should be essentially identical in features.

**Comment by xbony2** [comment]
Not so simple to do.

**Comment by elifoster** [comment]
Yes but it's very important. About half of the pe users are on ios

**Comment by kingbudderjr** [comment]
Ios system is very difficult to operate, thats why there arent that many
mods
Il 01/Set/2015 13:24, "Eli Clemente Gordillo Foster" <
notifications@github.com> ha scritto:

**Comment by xbony2** [comment]
There's no easy API, or at least one that is updated.

**Comment by elifoster** [comment]
Well when there is one, this should be of high priority.
